cmd/lotus/daemon.go

```diff
@@ -515,23 +537,19 @@ func ImportChain(ctx context.Context, r repo.Repo, fname string, snapshot bool)
 	}
 
 	bar.Start()
-	ts, err := cst.Import(ctx, ir)
+	ts, gen, err := cst.Import(ctx, ir)
 	bar.Finish()
 
 	if err != nil {
 		return xerrors.Errorf("importing chain failed: %w", err)
 	}
 
-	if err := cst.FlushValidationCache(ctx); err != nil {
-		return xerrors.Errorf("flushing validation cache failed: %w", err)
-	}
-
-	gb, err := cst.GetTipsetByHeight(ctx, 0, ts, true)
-	if err != nil {
-		return err
-	}
+	//if err := cst.FlushValidationCache(ctx); err != nil {
+	//return xerrors.Errorf("flushing validation cache failed: %w", err)
+	//}
```

Comment:
Pretty sure we want this

```suggestion
	if err := cst.FlushValidationCache(ctx); err != nil {
		return xerrors.Errorf("flushing validation cache failed: %w", err)
	}
```

(If we want to make it faster, this technically could run in parallel with the import)

Reply:
Good catch, thanks